website/docs/r/cdn_frontdoor_origin.html.markdown

```diff
@@ -0,0 +1,102 @@
+  azure_origin_id                       = ""
+
+  health_probes_enabled          = true
+  enforce_certificate_name_check = false
+  host_name                      = "contoso.com"
+  http_port                      = 0
+  https_port                     = 0
+  origin_host_header             = "www.contoso.com"
+  priority                       = 0
+  weight                         = 0
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Frontdoor Origin. Changing this forces a new Frontdoor Origin to be created.
+
+* `cdn_frontdoor_origin_group_id` - (Required) The ID of the Frontdoor Origin Group. Changing this forces a new Frontdoor Origin Group to be created.
+
+* `host_name` - (Required) The address of the origin. Domain names, IPv4 addresses, and IPv6 addresses are supported. This should be unique across all Frontdoor Origins in an Frontdoor Endpoints.
+
+* `cdn_frontdoor_origin_id` - (Optional) Resource ID.
+
+* `health_probes_enabled` - (Optional) Are health probes enabled against backends defined under the backendPools? Health probes can only be disabled if there is a single enabled backend in single enabled backend pool. Possible values are `true` or `false`. Defaults to `true`.
+
+* `enforce_certificate_name_check` - (Optional) Whether to enable certificate name check at origin level. Possible values are `true` or `false`. Defaults to `false`.
+
+* `http_port` - (Optional) The value of the HTTP port. Must be between `1` and `65535`. Defaults to `80`.
+
+* `https_port` - (Optional) The value of the HTTPS port. Must be between `1` and `65535`. Defaults to `443`.
+
+* `origin_host_header` - (Optional) The host header value sent to the origin with each request. Possible values include an `IPv4` IP address, `IPv6` IP address or a valid `domain name`. If you leave field undefined, the requests hostname determines this value. Azure Frontdoor Origins, such as Web Apps, Blob Storage, and Cloud Services require this host header value to match the origins hostname. This fields value overrides the host header defined in the Frontdoor Endpoint. For more information on how to properly set the origin host header value please see the [product documentation](https://docs.microsoft.com/azure/frontdoor/origin?pivots=front-door-standard-premium#origin-host-header).
+
+* `priority` - (Optional) Priority of origin in given origin group for load balancing. Higher priorities will not be used for load balancing if any lower priority origin is healthy.Must be between `1` and `5`(inclusive). Defaults to `1`.
+
+* `weight` - (Optional) Weight of the origin in given origin group for load balancing. Must be between `1` and `1000`(inclusive). Defaults to `500`.
+
+---
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported:
+
+* `id` - The ID of the Frontdoor Origin.
+
+* `deployment_status` - 
```

Comment:
missing docs,

Reply:
also we typically do not expose these properties?

Reply:
Yep... I just overlooked it and forgot to remove it... doh! Fixed.